application/lib/service/event_database_service.dart

```diff
@@ -185,4 +185,60 @@ class EventDatabaseService {
       throw Exception(e.message);
     }
   }
+
+  Future<void> addRating(String userId, String eventId, double rating) async {
+    try {
+      DocumentReference userRef = _firestore.collection('users').doc(userId);
```

Comment:
az értékeléseket nem szerencsés közvetlenül a user táblában tárolni. 
Egy külön tábla kéne legyen a db-ben a ratings. Vegyük azt az esetet, hogy törölni vagy módosítani kell egy értékelést a usernél. Ekkor ha egy listában van, akkor hozzá kell nyúlni az egész listához. Ez plusz futási idő + benne van a lehetőség hogy véletlen másik értékelést módosítsunk. 
Ugyan ez igaz a likeokra is majd. 
Ilyen több-a-többhöz esetekben szoktunk kapcsolótáblát használni. Adatbázis órán biztos tanultatok róla.